minishift-ci-index.yaml

```diff
@@ -75,3 +75,8 @@
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
           timeout: '20m'
+      - '{git_repo}':
```

Comment:
@hferentschik , right now waiting for new a/c, once that is done we can update that. 

Ya will add lala and praveen in admin list too.